proto/event.proto

```diff
@@ -50,17 +63,53 @@ message Histogram {
   map<string, string> tags = 5;
 }
 
+message AggregatedSet {
+  string name = 1;
+  repeated string values = 2;
+  google.protobuf.Timestamp timestamp = 6;
+  map<string, string> tags = 7;
+}
+
+message AggregatedDistribution {
+  string name = 1;
+  repeated double values = 2;
+  repeated uint32 sample_rates = 3;
+  google.protobuf.Timestamp timestamp = 6;
+  map<string, string> tags = 7;
+}
+
+message AggregatedHistogram {
+  string name = 1;
+  repeated double buckets = 2;
+  repeated uint32 counts = 3;
+  uint32 count = 4;
```

Comment:
Is this just the sum of `counts` above? We could maybe omit it and just calculate on demand.

Reply:
Not really. 'Buckets' are less-or-equal boundaries, and `count` field represents extra le='+Inf' bucket, which is implicit. I'm not sure if it's universal, but it's a mandatory bucket in Prometheus.